src/mon/OSDMonitor.cc

```diff
@@ -4764,9 +4764,17 @@ void OSDMonitor::check_osdmap_subs()
 void OSDMonitor::check_osdmap_sub(Subscription *sub)
 {
   dout(10) << __func__ << " " << sub << " next " << sub->next
-	   << (sub->onetime ? " (onetime)":" (ongoing)") << dendl;
+	   << (sub->onetime ? " (onetime)":" (ongoing)")
+	   << (sub->want_latest ? " (noosd)":" (osd)")
```

Comment:
name it as ''latest'

Reply:
all frag should in single (). such as (onetime, latest)

Reply:
updated it